lang.lua

```diff
@@ -74,11 +76,15 @@ table.sort(options, function(a,b) return b[2] < a[2] end)
 
 -- Match the best language we got
 for index, lang in pairs(options) do
-  if langs[lang[1]] then
-    return lang[1]
+  for value, langavail in pairs(langs) do
+    if langavail:lower() == lang[1] then
```

Comment:
Rather than lower casing each language entry, I would just lower case the input string above (and also replace ```_``` with ```-``` or vice versa) so that a table look up will work and no iteration has to happen.